src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -740,6 +806,219 @@ def test_get_job_info(self):
             infos, {id: ALL_RESPONSE_DATA[MESSAGE_TYPE["INFO"]][id] for id in ALL_JOBS}
         )
 
+    @mock.patch(CLIENTS, get_mock_client)
+    def test_add_errors_to_results__concat_errs__integrated(self):
```

Comment:
Test concat errors (i.e., in the case that `jm.cancel_jobs` needs to concat an error from an output state from `jm._construct_output_state_set`